src/Appwrite/Utopia/Response/Model/Specification.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Specification extends Model
+{
+    public function __construct()
+    {
+
+        $this->addRule('memory', [
+            'type' => self::TYPE_INTEGER,
+            'description' => 'Memory size in MB.',
+            'default' => 0,
+            'example' => 512
+        ]);
+        $this->addRule('cpus', [
+            'type' => self::TYPE_INTEGER,
```

Comment:
This should be a float now too, let's add a test covering it too

```suggestion
            'type' => self::TYPE_FLOAT,
```